automatic/7zip.install/update.ps1

```diff
@@ -0,0 +1,29 @@
+. "$PSScriptRoot\..\7zip\update.ps1"
+
+$softwareNamePrefix = '7-zip'
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -FileNameBase '7zip'
+  $Latest.ChecksumType = 'sha256'
+}
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(?i)(^\s*softwareName\s*=\s*)'.*'" = "`$1'$softwareNamePrefix $($Latest.RemoteVersion)*'"
```

Comment:
Actually, in this case it will be accurate.
The software name should be part of or equal to the name displayed in program and features